core/domain/skill_domain.py

```diff
@@ -336,6 +344,10 @@ def __init__(
                 created.
             last_updated: datetime.datetime. Date and time when the
                 skill was last updated.
+            superseding_skill_id: str. skill Id of the skill we
+                merge this skill into.
```

Comment:
... This is only non-None if this skill has been marked as a duplicate of another one.